src/Commands/CommandFactory.cs

```diff
@@ -87,6 +89,20 @@ private void RegisterCommandGroup()
         RegisterMcpServerCommands();
     }
 
+    private void RegisterBestPracticesCommand()
+    {
+        // Register Azure Best Practices command at the root level
+        var bestPractices = new CommandGroup(
```

Comment:
We currently do not support commands with hyphen because hypens are removed when we print the command.  Run tool list command to see the command outputted.

Reply:
removed hyphen. 
with hypen, list tools shows azmcp best practices